code/standards/checkstyle.xml

```diff
@@ -75,13 +83,21 @@
     <module name="FinalClass"/>
     <module name="HideUtilityClassConstructor"/>
     <module name="InterfaceIsType"/>
-    <module name="VisibilityModifier"/>
+    <module name="VisibilityModifier">
+      <property name="protectedAllowed" value="true" />
+      <property name="allowPublicFinalFields" value="true" />
```

Comment:
Is there something I can do to avoid this for now? Perhaps leave the exception out, and add supressions when we're ready to actually error out? I don't want to hold off on the entire diff due to a single rule :/